AutoRest/TestServer/swagger/httpInfrastructure.json

```diff
@@ -2795,6 +2795,9 @@
       }
     },
     "A": {
+      "x-ms-exception": {
```

Comment:
Why not name `A` as `MyException`? Do you have a strong use case for custom exception names?